src/modules/tasks.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable import/extensions */
+import { renderTasks } from './loadTasks.js';
+
+const listTarget = document.querySelector('.list');
+
+const addInputField = document.getElementById('inputTask');
+
+export const editAddTask = () => {
+  listTarget.addEventListener('keypress', (e) => {
+    const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+    if ((e.key === 'Enter') && (e.target.matches('.edit-input'))) {
+      const idArray = (e.target.id).split('_');
+      const taskId = Number(idArray[idArray.length - 1]);
+      for (let i = 0; i < tasks.length; i += 1) {
```

Comment:
what if you replace the `for` loop with a `forEach` loop, it is more elegant

Reply:
Yes a foreach can also get the job done